apps/project/migrations/rename_dublicate_project_name.py

```diff
@@ -0,0 +1,34 @@
+
+from django.db import migrations
+from django.db.models import Count
+
+
+def _rename_dublicate_name(Project):
+    dublicate_projects = Project.objects.values('title').annotate(Count('id')).order_by().filter(id__count__gt=1)
+    dublicate_project_dict = {}
+    for project in dublicate_projects:
+        p = {project['title']: Project.objects.filter(title=project['title'])}
+        dublicate_project_dict.update(p)
+    for key, project in dublicate_project_dict.items():
+        for index, p in enumerate(project, start=1):
+            p.title = p.title + str(index)
```

Comment:
Title should be "<project-name> (index)" cc: @AdityaKhatri

Reply:
@AdityaKhatri